Project.toml

```
@@ -15,7 +15,7 @@ rr_jll = "e86bdf43-55f7-5ea2-9fd0-e7daa2c0f2b4"
 
 [compat]
 AWS = "1.49"
```

I believe this can be set to the suggestion below as this had nothing special in it.

```suggestion
AWS = "1"
```

---

For reference, I added the compat in #41, with no actual clue of the version needed :slightly_smiling_face:

---

Almost all version updates to `AWS.jl` are around re-generating definitions from new versions off of the AWS JS SDK.
They may or may not include some bug fixes (although usually those are coming in as patches), or more rarely some major-ish reworks that work backwards.